deps/rabbitmq_management/priv/schema/rabbitmq_management.schema

```diff
@@ -528,3 +528,10 @@ fun(File) ->
     ReadFile = file:list_dir(File),
     element(1, ReadFile) == ok
 end}.
+
+%% Disables setting operator policies over API.
+
+{mapping, "management.disable_operator_policy_edit", "rabbitmq_management.disable_operator_policy_edit", [
```

Comment:
I'd use `management.restrictions.operator_policy_changes.disabled` and `rabbitmq_management.restrictions.operator_policy_changes.disabled`. It'd be a bit more annoying to access this nested proplist but you'd have to do it only once in one predicate function.